test/test_utils.jl

```diff
@@ -1,23 +1,65 @@
 @testset "test_utils.jl" begin
 
-    @testset "FakeTemplate{PointEstimate, SingleOutput}" begin
-        temp = FakeTemplate{PointEstimate, SingleOutput}()
+    @testset "FakeTemplate{PointEstimate, SingleOutput, PointPredictInput}" begin
+        temp = FakeTemplate{PointEstimate, SingleOutput, PointPredictInput}()
+        test_interface(temp)
+
+        temp = FakeTemplate{PointEstimate, SingleOutput, PointPredictInput}()
+        test_interface(temp; inputs=[rand(5), rand(5)], outputs=rand(1, 2))
+    end
+
+    @testset "FakeTemplate{PointEstimate, MultiOutput, PointPredictInput}" begin
```

Comment:
Can we do these ones in a loop? Something like
```julia
@testset "FakeTemplate{R, S, T}"  for R in (...), S in (...), T in (...)
    temp = FakeTemplate{R, S, T}()
    test_interface(temp)
end
```
(although I guess that doesn't account for the special case in the previous one)